cura/CuraApplication.py

```diff
@@ -961,6 +960,7 @@ def registerObjects(self, engine):
         qmlRegisterType(BrandMaterialsModel, "Cura", 1, 0, "BrandMaterialsModel")
         qmlRegisterType(MaterialManagementModel, "Cura", 1, 0, "MaterialManagementModel")
         qmlRegisterType(QualityManagementModel, "Cura", 1, 0, "QualityManagementModel")
+        qmlRegisterType(MachineManagementModel, "Cura", 1, 0, "MachineManagementModel")
```

Comment:
This is new in this Cura, so the version number should be higher. Check the other grouped version numbers (below in this method) as example.

Reply:
All the models introduced in the MM refactor are also new and all of them have the same version number. I agree, but then we should change them all.